app/controllers/rescue_codes_controller.rb

```diff
@@ -0,0 +1,33 @@
+class RescueCodesController < ApplicationController
+  before_action :confirm_two_factor_authenticated
+
+  def new
+    @codes = current_user.create_backup_codes
```

Comment:
I'm not sure creating backup codes should be available via a GET request. Do we want to allow a user to regenerate codes by simply refreshing the page? Perhaps a button that the user has to click (POST request) would be better?